Xamarin.Forms.Mocks.Tests/AppThemeTests.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Xamarin.Forms;
+
+namespace Xamarin.Forms.Mocks.Tests
+{
+    [TestFixture]
+    public class AppThemeTests
```

Comment:
It looks like `Application.Current` can't be null or these tests will throw an NRE:

https://github.com/xamarin/Xamarin.Forms/blob/5a8858277ad381d055cdcb0fa6634bfc684c30b1/Xamarin.Forms.Core/OnAppTheme.cs#L54-L56

Reply:
Here is a decent example of setting `Application.Current`: https://github.com/jonathanpeppers/Xamarin.Forms.Mocks/blob/4a372b86366c52e2aeecb94535ae469841f9a7cd/Xamarin.Forms.Mocks.Tests/ApplicationTests.cs#L6-L44

Reply:
did this from VS Code... for some reason dotnet test wasn't giving me outputs I'll update the test